packages/faustwp-core/package.json

```diff
@@ -5,6 +5,18 @@
   "main": "dist/cjs/index.js",
   "module": "dist/mjs/index.js",
   "types": "dist/cjs/index.d.ts",
+  "exports": {
+    ".": {
+      "require": "./dist/cjs/index.js",
+      "import": "./dist/mjs/index.js"
+    },
+    "./toolbar": {
+      "require": "./dist/cjs/components/Toolbar/index.js",
+      "import": "./dist/mjs/components/Toolbar/index.js"
+    },
+    "./dist/css/toolbar.css": "./dist/css/toolbar.css",
```

Comment:
Instead of exporting the style, we should include it outselves in the toolbar component